Guide/src/dev_guide/getting_started/build_openvmm.md

```diff
@@ -88,3 +88,26 @@ Note: the specific package that throws this error may vary, and may not always b
 You attempted to build OpenVMM without first restoring necessary packages.
 
 Please run `cargo xflowey restore-packages`, and try again.
+
+### use of unstable library feature 'absolute_path'
+
+**Error:**
+
+```
+error[E0658]: use of unstable library feature 'absolute_path'
+  --> flowey/flowey/src/lib.rs:37:17
+   |
+37 |                 std::path::absolute(self)
+   |                 ^^^^^^^^^^^^^^^^^^^
+   |
+   = note: see issue #92750 <https://github.com/rust-lang/rust/issues/92750> for more information
+
+For more information about this error, try `rustc --explain E0658`.
+error: could not compile `flowey` (lib) due to previous error
+```
+
+**Solution:**
+
+OpenVMM requires a more recent version of Rust than the version you have installed.
+
+Please uninstall the version of Rust you are using and use [rustup](https://rustup.rs/) to download the latest stable release of Rust.
```

Comment:
Lets swap this entire line with links back to the Windows / Linux sections on "Installing Rust" (i.e: https://openvmm.dev/dev_guide/getting_started/linux.html#installing-rust / https://openvmm.dev/dev_guide/getting_started/windows.html#installing-rust)